lib/access_lint/rails/engine.rb

```diff
@@ -5,7 +5,7 @@ class Engine < ::Rails::Engine
 
       initializer "access_lint-rails" do |app|
         app.config.assets.precompile << "accesslint.js"
-
+        app.config.filter_parameters << :accesslint
```

Comment:
Hey @ckundo I think just for the host app. It's probably a plus to make it possible for anyone running the engine to make their own choice about how verbose they want their logs to be.